closure/goog/net/jsloader_test.js

```diff
@@ -134,7 +138,24 @@ function testLoadMany() {
 
     // Check that the 1st test was already loaded.
     assertEquals('verification object', 'Test #1 loaded', window.test1);
+
+    // all scripts loaded - verify that result has fired
+    assertTrue('deferred has fired', result.hasFired());
+
+    // on last script loaded, set variable
+    window.test4 = msg;
+
+    testCase.waitForAsync('testLoadMany');
   };
+
+  result.addCallback(function() {
+    testCase.continueTesting();
+
+    // verify that the last loaded script callback has executed
+    assertEquals('verification object', 'Test #4 loaded', window.test4);
+  });
+
+
```

Comment:
which ones? I've copied the writing style in that file, they're all over the place